src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -137,11 +161,22 @@ export default {
         position: namePosition,
       }]);
     },
+    createDefaultFlowMarker() {
+      const isDefault = this.isDefaultFlow();
+      this.shape.attr('line', {
+        sourceMarker: {
+          'type': 'polyline',
+          'stroke-width': isDefault ? 2 : 0,
+          points: '2,6 6,-6',
+        },
```

Comment:
This could be at the heart of the method that you have instead of the computed property. I'd also suggest inlining the `isDefault` const.

Reply:
Fixed